docs/src/pom.md

```diff
@@ -196,6 +132,8 @@ await expect(playwrightDev.tocList).toHaveText([
 ]);
 ```
 
+Instanciating the different Page Objects can be cumbersome and makes the test code unnecessarily complex. [Test fixtures](./test-fixtures) are a good way to establish the environment for each test, they can be used to initialize Page Object. See how to [create a fixture](./test-fixtures#creating-a-fixture) with a Page Object.
```

Comment:
- typo: Instan_c_iating
- I don't like how this insists on POMs being cumbersome and complex. That's an opinion, and we'd like our users to form their own opinions. Can we rephrase in the vein of "take a look at how to create a fixture with a Page Object, perhaps you'll find it more convenient"?

Reply:
Understood. I'm probably biased.

For the moment, I'll just stick to removing the JS examples (like the rest of docs). Maybe POM with fixtures could be an article or something.